src/util_misc.cpp

```diff
@@ -657,13 +657,13 @@ void Util::allowGdbDebug(int currentDebugLevel)
   if (Util::isValidFd(PROTECTED_DEBUG_SOCKET_FD)) {
     int requestedDebugLevel = 0;
     // Inform parent of current level
-    write(PROTECTED_DEBUG_SOCKET_FD,
-          &currentDebugLevel,
-          sizeof(currentDebugLevel));
+    JASSERT(write(PROTECTED_DEBUG_SOCKET_FD,
+                  &currentDebugLevel,
+                  sizeof(currentDebugLevel)) != -1);
     // Read the requested level from the parent
-    read(PROTECTED_DEBUG_SOCKET_FD,
-         &requestedDebugLevel,
-         sizeof(requestedDebugLevel));
+    JASSERT(read(PROTECTED_DEBUG_SOCKET_FD,
+                 &requestedDebugLevel,
+                 sizeof(requestedDebugLevel)) == -1);
```

Comment:
Should this be `!= -1`?

Reply:
Thanks.   Working late at night.  I'll fix it.